cmd/monitor/streaming/bootstrap.yaml

```diff
@@ -189,20 +189,6 @@ log-storage-elasticsearch:
   write_timeout: "1m"
   index_type: "logs"
 
-log-persist:
-  _enable: ${WRITE_LOG_TO_ES_ENABLE|WRITE_LOG_TO_CLICKHOUSE_ENABLE:true}
-  input:
-    topics: "${LOG_TOPICS:spot-container-log,spot-job-log}"
-    group: "${LOG_GROUP_ID:erda-logs-dev}"
-    offsets:
-      initial: earliest
-  id_keys: "${LOG_ID_KEYS:TERMINUS_DEFINE_TAG,terminus_define_tag,MESOS_TASK_ID,mesos_task_id}"
-  read_timeout: "5s"
-  buffer_size: ${LOG_BATCH_SIZE:4096}
-  parallelism: ${LOG_PERSIST_PARALLELISM:6}
```

Comment:
Where to set parallelism in pipeline mode, is `EXPORTER_CH_LOG_CURRENCY_NUM`?

Reply:
that's it